lib/twitter_cldr/js/mustache/data_bundle.coffee

```diff
@@ -0,0 +1,38 @@
+###
+// Copyright 2012 Twitter, Inc
+// http://www.apache.org/licenses/LICENSE-2.0
+// TwitterCLDR (JavaScript) v{{version}}
+// Authors:     Cameron Dutro [@camertron]
+                Kirill Lashuk [@KL_7]
+                portions by Sven Fuchs [@svenfuchs]
+// Homepage:    https://twitter.com
+// Description: Provides date, time, number, and list formatting functionality for various Twitter-supported locales in Javascript.
+###
+
+{{#source_map}}
+###
+//@ sourceMappingURL={{locale}}.map
+###
+{{/source_map}}
+
+# required alias definition that adds twitter_cldr to Twitter's static build process
+###<<module_def>>###
+
+
+TwitterCldrDataBundle =
+{{{contents}}}
+
+root = if exports?
+  exports
+else
+  this.TwitterCldrDataBundle = {}
+  this.TwitterCldrDataBundle
+
+root[key] = obj for key, obj of TwitterCldrDataBundle
+
+if this.TwitterCldr?
+  try
```

Comment:
How about instead of using try-catch we just check manually if the data is set? I feel like calling our own method just to catch an exception from it is not very efficient. Everywhere else we should still use `get_data` just so we get a nice error if it's not set, but here I feel like it's alright to just use `TwitterCldr.data` directly and then set it via `set_data` if necessary.